main.py

```diff
@@ -1,10 +1,64 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
```

Comment:
Change the order of your imports according to `pep8`. Here `json` should be first.